src/main/app/wmt-mfs-crm-exp.xml

```diff
@@ -29,35 +29,64 @@ http://www.springframework.org/schema/beans http://www.springframework.org/schem
         <exception-strategy ref="wmt-mfs-crm-exp-apiKitGlobalExceptionMapping" doc:name="Reference Exception Strategy"/>
     </flow>
     <flow name="wmt-mfs-crm-exp-console">
-        <http:listener config-ref="wmt-mfs-domain-httpListenerConfig" path="/mfs-crm/console/*" doc:name="HTTP"/>
-        <apikit:console config-ref="wmt-mfs-crm-exp-config" doc:name="APIkit Console"/>
+  		<http:listener config-ref="HTTP_Listener_ConfigurationTemp" path="/mfs-crm/console/*" doc:name="HTTP"/>
```

Comment:
Please review this http listner config ref, it should use the wmt-mfs-domain one.